apps/site/assets/ts/stop/components/StopMapRedesign.tsx

```diff
@@ -26,7 +32,7 @@ const StopMapRedesign = ({ stop }: Props): ReactElement<HTMLElement> => {
         tooltip: null
       } as MapMarker
     ],
-    polylines: [],
+    polylines: uniqBy(lines, "id"),
```

Comment:
question: should this filtering also be happening on the server side?

Reply:
The merging-shapes-from-many-routes thing only happens in the frontend (all routes are fed into this map), so it makes more sense to de-duplicate here.